cmd/cluster-cloud-controller-manager-operator/main.go

```diff
@@ -64,7 +64,10 @@ func init() {
 }
 
 func main() {
-	flag.Set("logtostderr", "true")
+	if err := flag.Set("logtostderr", "true"); err != nil {
```

Comment:
Will constantly fail if `-logtostderr` flag not set. CI failing because of this as well.

Reply:
For `render` command its the same.

Can we to remove this block? `klog` put logs to stderr by default anyway.